package-lock.json

```diff
@@ -48,7 +48,7 @@
         "starknet": "^5.25.0",
```

Comment:
I suggest you to remove changes made to package-lock.json, maybe also package.json (was the three update really necessary ?), because it creates conflicts.